dvc/output/local.py

```diff
@@ -18,6 +18,13 @@ class OutputLOCAL(OutputBase):
     REMOTE = RemoteLOCAL
     sep = os.sep
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if self.repo and path_isin(self.def_path, self.repo.root_dir):
+            self.def_path = relpath(self.def_path, self.repo.root_dir)
+            self.path_info = self._parse_path(self.remote, self.def_path)
```

Comment:
Tried to move from `base` to `local` but not sure about the distinction between `def_path` and `path_info`

Reply:
second line not required - already resolved by `parse_path` - so removed